cibuildwheel/__main__.py

```diff
@@ -290,6 +292,10 @@ def main() -> None:
     else:
         assert_never(platform)
 
+    final_contents = set(output_dir.iterdir())
+    new_contents = final_contents - existing_contents
+    print("Produced wheels:", *sorted(f.name for f in new_contents))
```

Comment:
2 suggestions:
- Can we make this foldable, the folded thing just showing "Produced N wheels"?
- Maybe one wheel per line, `ls` style? Especially if this is foldable, this will be more clear to look up which wheel is missing, I'd expect.

Reply:
We could, I considered that. We don't' collapse the header, though, so was thinking this might be best uncollapsed too. But I was on the fence.

Reply:
But this is something you don't always want to see? I my case, I know e.g. that I'm expecting (_looks at my releases_) 32 wheels. So if that's fine, without an error, I feel like that's more clear than always seeing everything.
At the very least, everything on one line loses all oversight, for me, when I checked the example build.

Reply:
Yes, I'll collapse and do one per line. Though, if you scroll all the way down the log to the end anyway, not sure you should then have to uncollapse it - there's nothing past it, after all, in that step, usually. :)

Reply:
OK! Hmmm, indeed, I thought GHA would scroll down for me, but not really.

But I switched around my reasoning, it seems. I first thought of 2., to see the different wheel names nicely lined up underneath each other (also catches outliers better, if there's files that shouldn't be there!), and only then thought about 1. since it would be too long and would need to get collaped :-)

Reply:
One-per-line seems like a good idea. Maybe indent it too so they're under the header? like

```
16 wheels produced:
  spam-0.1.0-cp27-manylinux1_x86_64.whl
  spam-0.1.0-cp35-manylinux2010_x86_64.whl
  spam-0.1.0-cp35-manylinux1_x86_64.whl
  spam-0.1.0-cp35-manylinux2010_x86_64.whl
  spam-0.1.0-cp36-manylinux1_x86_64.whl
  spam-0.1.0-cp36-manylinux2010_x86_64.whl
  spam-0.1.0-cp37-manylinux1_x86_64.whl
  spam-0.1.0-cp37-manylinux2010_x86_64.whl
  spam-0.1.0-cp38-manylinux1_x86_64.whl
  spam-0.1.0-cp38-manylinux2010_x86_64.whl
  spam-0.1.0-cp39-manylinux1_x86_64.whl
  spam-0.1.0-cp39-manylinux2010_x86_64.whl
  ...
```

Unsure about collapsing, I see the argument, but I feel like in a success case the actual output of all the logs above seems important enough to be unhidden by default.

Reply:
Yes, and the overall step is always collapsed in all CI's, so collapsing inside the step wouldn't help anything.